### PR TITLE
docs: update referenced freebsd version

### PR DESCRIPTION
**Pull request by remyabel**
Fixes: https://github.com/borgbackup/borg/issues/5942

### CODE REVIEW

**Comment by ThomasWaldmann** [comment]
no, we can't do it like that:
- changes.rst should be readable in source form also, so there is no placeholder expansion
- entries in changes.rst (esp. older ones) are not supposed to change (when updating the placeholder definition)

**Comment by remyabel** [comment]
Hmm OK, I'll redo it.

**Comment by remyabel** [comment]
OK if I understood correctly, all I needed to do was revert the changes to the changelog. Couldn't find any other instances that needed to be updated.

File: docs/installation.rst [hunk 1]
```diff
@@ -113,7 +113,7 @@ are available on the releases_ page for the following platforms:
 * **MacOS**: 10.10 or newer (To avoid signing issues download the file via
   command line **or** remove the ``quarantine`` attribute after downloading:
   ``$ xattr -dr com.apple.quarantine borg-macosx64.tgz``)
-* **FreeBSD**: 10.2 (unknown whether it works for older releases)
+* **FreeBSD**: |freebsd_version| (unknown whether it works for older releases)
```
Comment (ThomasWaldmann) [inline docs/installation.rst:116]: if this is only used at a single place, guess it is easier to just have it literally there, not using a variable.

File: docs/installation.rst [hunk 2]
```diff
@@ -113,7 +113,7 @@ are available on the releases_ page for the following platforms:
 * **MacOS**: 10.10 or newer (To avoid signing issues download the file via
```
Comment (ThomasWaldmann) [inline docs/installation.rst:113]: this also needs fixing, it is macos sierra 10.12 now.

**Comment by ThomasWaldmann** [comment]
Thanks!

**Comment by ThomasWaldmann** [comment]
Can you check same for 1.1-maint (binary build OS versions see Vagrantfile there)?

**Pull request merged**